src/console/CmdProcessor.cpp

```diff
@@ -381,6 +382,14 @@ bool CmdProcessor::process(folly::StringPiece cmd) {
     return true;
 }
 
+void CmdProcessor::normalize(folly::StringPiece &command) {
+    command  = folly::trimWhitespace(command);
+    if (command.endsWith(";")) {
```

Comment:
Take care of multiple ending `;`s, e.g. `quit;;`

Reply:
Nice, I will using `while` to determine the loop ending.